frontend/src/pages/Users/Users.js

```diff
@@ -145,17 +163,25 @@ function UserList(props) {
           filteringAriaLabel="Filter users"
         />
       }
-    />
+    /></>
   );
 }
 
 export default function Users() {
   const users = useSelector(selectUserIndex);
   const navigationOpen = useState(['app', 'sidebar', 'drawerOpen']);
+  const usernamePath = ['app', 'users', 'email'];
+  const username = useState(usernamePath);
   const refreshUsers = () => {
     ListUsers();
   }
 
+  const createUser = () => {
+    let user = {Username: username};
+    CreateUser(user);
```

Comment:
Probably makes sense to put the `clearState` into a success callback for the `CreateUser` function. That way if there's an error (like the user already existing) then the input from the user doesn't get cleared out.

Reply:
done